src/test/java/be/seeseemelk/mockbukkit/tags/internal/InternalTagParserTest.java

```diff
@@ -24,12 +24,6 @@ void insertInternalTagValues_registries()
 		assertFalse(InternalTag.NON_SOLID_BLOCKS.getValues().isEmpty());
 	}
 
-	@Test
-	void checkTagged_fromTag()
```

Comment:
Was this tag removed from the game?

Reply:
Because I replaced the data with automatically generated data, which does not use tags.